examples/tcp-sockets/tcp-sockets.go

```diff
@@ -0,0 +1,173 @@
+// Socket programming allows direct communication between
```

Comment:
There's no need for these two long paragraphs explaining what TCP is. Let's assume the reader knows, and just wants to learn how to do this in Go